src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@
 #
 
 # Global
-#github.token=<your key here or in the secret>
+github.token=${github.token}
```

Comment:
You don't really need this. You can pass the `github.token` via system properties or env vars anyway.

Reply:
I mean that you don't need filtering.
However, its a matter of preference, so I don't really mind.